sot/opcode_translator/executor/dispatcher.py

```diff
@@ -101,12 +134,29 @@ def register(
         kwtypes: dict[str, str],
         handler: Callable[..., Any],
     ):
+        """
+        Registering function signatures on handlers
```

Comment:
```suggestion
        Registering function signature.
```

`on handlers` 是实现细节，删掉就好